GpsDataCapturer/wear/src/main/res/layout/activity_gps_main.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.wear.widget.BoxInsetLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/dark_grey"
+    android:padding="@dimen/box_inset_layout_padding"
+    tools:context=".GpsMainActivity"
+    tools:deviceIds="wear">
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:padding="@dimen/inner_frame_layout_padding">
+
+        <Button
+            android:id="@+id/start_button"
+            android:layout_width="72dp"
+            android:layout_height="wrap_content"
+            android:onClick="onStartButtonClick"
+            android:layout_alignParentStart="true"
+            android:layout_alignParentBottom="true"
+            android:layout_marginStart="@dimen/screen_percentage_10"
+            android:layout_marginBottom="@dimen/screen_percentage_15"
+            android:text="@string/start_capture"
+            android:textSize="@dimen/screen_percentage_05" />
+
+        <Button
```

Comment:
Do we need 2 buttons?
We can use single button and toggle text based on current status.

Reply:
Right, will do that.